Tests/SentryTests/Performance/TransactionGeneratorTests.swift

```diff
@@ -0,0 +1,132 @@
+import XCTest
+
+/**
+* This isn't an actual test. It creates transactions and sends them to the Sentry, but doesn't verify if they arrive there.
+*/
+class TransactionGeneratorTests: XCTestCase {
```

Comment:
What about creating a README.md in the `Tests` directory where we can mention such tests?
I'm afraid it might become forgotten or simply folks getting involved in the code base later will simply not know about the existence of this code.

Reply:
Good point, done.